android/app/src/main/res/values-ar/strings.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
+  <!-- Channel label for notifications shown in App > Notifications > Path Check BT -->
+  <string name="notification_channel_description">إشعارات PathCheck BT</string>
```

Comment:
We no longer need this string, it's not being used

Reply:
@amandabeiner let me know when you're back at this and we can go on a lokalise journey to clean up some stuffffff